database/index.js

```diff
@@ -0,0 +1,8 @@
+const mysql = require('mysql');
+const mysqlConfig = require('./config.js');
+
+const db = mysql.createConnection(mysqlConfig);
+
+const getAllPhotos = () => {
+  let queryStr = 'select '
+}
```

Comment:
You should install a linter immediately so that it can tell you about issues like "you're missing a semicolon" instead of relying on your PR reviewers to catch these mistakes for you.